packages/v4/src/content/design-guidelines/components/notification-drawer/notification-drawer.md

```diff
@@ -4,34 +4,32 @@ section: components
 related: ['Notification drawer', 'Notification badge', 'Drawer']
 ---
 
-The **Notification Drawer** uses the [Drawer component](/documentation/core/components/drawer) to provide a place for viewing and managing incoming notifications.
+The Notification drawer is a place where users can view and manage notifications that are generated by the system. It allows them to see a list of notifications without the need to navigate away from the current screen.
 
-The notification drawer is a place where users can view and manage asynchronous notifications that are generated by the system. It allows them to see a list of notifications without the need to navigate away from the current screen.
-
-## Notification drawer elements
+## Elements
 
 ### Basic notification drawer
 
 <img src="./img/Simple-Notification-Drawer.png" alt="basic notification drawer" width="743"/>
 
-**1. Header:** The header contains the title of the drawer (default to “Notifications”).
-
-**2. Unread count (optional):** If unread notifications exist, report the number of unread notifications. Otherwise leave blank.
+1. **Header:** The header contains the title of the drawer (default to “Notifications”).
 
-**3. Global actions (optional):** This menu contains actions that can be applied against all notifications in the drawer. The following are standard menu items. This menu may be added according to the needs of your application.
+2. **Unread count (optional):** If unread notifications exist, report the number of unread notifications. Otherwise leave blank.
 
+3. **Global actions (optional):** This menu contains actions that can be applied against all notifications in the drawer. The following are standard menu items. This menu may be added according to the needs of your application.
+  - **Mark all read** - marks all notifications in the drawer as read.
```

Comment:
These bullets under #3 should be indented in one more, so that it's not inline with the bulleted list of possible!